index.html

```diff
@@ -401,7 +401,7 @@
     </div>
 
     <script src="lib/js/head.min.js"></script>
-    <script src="palantirnet-theme/assets/js/reveal.js"></script>
+    <script src="js/reveal.js"></script>
 
```

Comment:
Ah, I see.  I had expected to just pull a fresh Reveal each time (and/or copy from a previous slide deck.)  Don't we need things like the plugin directory and Reveal's own CSS, etc?